frontend/javascripts/test/global_mocks.ts

```diff
@@ -0,0 +1,120 @@
+// Add more essential global mocks
+import { vi } from "vitest";
+
+import { annotationProto as SKELETON_ANNOTATION_PROTO } from "./fixtures/skeletontracing_server_objects";
+import { annotationProto as VOLUME_ANNOTATION_PROTO } from "./fixtures/volumetracing_server_objects";
+
+// Mock global objects
+global.performance = {
+  ...global.performance,
+  now: vi.fn(() => Date.now()),
+};
+
+// Mock common utility functions
+// Mock modules instead of using mockRequire
+vi.mock("libs/keyboard", () => ({
+  default: {
+    bind: vi.fn(),
+    unbind: vi.fn(),
+    withContext: (_arg0: string, arg1: () => void) => arg1(),
+  },
+}));
+
+vi.mock("libs/toast", () => ({
+  default: {
+    error: vi.fn(),
+    warning: vi.fn(),
+    close: vi.fn(),
+    success: vi.fn(),
+  },
+}));
+
+vi.mock("libs/window", async (importOriginal) => {
+  return {
+    ...(await importOriginal<typeof import("libs/window")>()),
+  };
+});
```

Comment:
this mocks libs/window only to import it again and expose it? is this really necessary or can it be removed? if necessary, a comment would help.